### ISSUE TITLE
Switch queries to update delivery stats UI to read replicas

### ISSUE THREAD

**Issue by syan-syan** [opengovsg/postmangovsg#1321]
Better isolation of other ops from large campaigns

**Comment by lamkeewei** [comment]
**tl;dr** I think statistics queries should remain on the primary database instance as there isn't a good solution to deal with query cancellation. Statistics queries are ran either on the `ops` or `statistics` table. Both tables should be small and fairly efficient to query.

____

**Problem**
While the campaign is sending, progress/stats are calculated by querying the ops table. If we were to use the read replica for this query, there might be situations where the query is cancelled when the rows on the primary database instance are updated or deleted (such as when the campaign is sent and is going to be logged). 

**Solutions**
The possible solutions to this are:
1. Add a retry for cancelled queries. Even with this the query might not always succeed especially if the rows are updated rapidly. 
2. Enable `hot_standby_feedback` on the read replica. This allows the read replica instance to send feedback about queries that it is currently executing to the primary instance. This would postpone `VACUUM` on the related tables on the primary database instance until the read queries on the replica have completed. Downside of this is that the tables will get bloated if there isn't a time to run `VACUUM`. Also, this setting is instance wide and cannot be limited to a specific table.

**Conclusion**
Given that both of these solutions have downsides, I think it's best to keep the query for campaign statistics on the primary database instance. Moreover, while the campaign is sending, statistics are derived from querying the ops table. This table should be many times smaller than the messages table and would not be a hog in terms of resources.
